pkgs/development/python-modules/rarfile/default.nix

```
@@ -8,13 +8,13 @@ assert !useUnrar -> libarchive != null;
 
 buildPythonPackage {
```

this is missing a rec, but would otherwise be correct
```suggestion
buildPythonPackage rec {
```